.github/stale.yml

```diff
@@ -0,0 +1,25 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+  - accessibility
+  - "breaking change :boom:"
+
+# Label to use when marking as stale
+staleLabel: wontfix
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed in 30 days if no further activity occurs.
```

Comment:
Do you think there is value in saying the duration before an issue is marked as stale so that users know when that happens?

> This issue has been automatically marked as stale because it has not had _activity in the last 60 days_. It will be closed in 30 days if no further activity occurs.

Reply:
@mcoker sounds good - just pushed up this change